src/app-root.ts

```diff
@@ -257,13 +262,34 @@ export class AppRoot extends LitElement {
       display: block;
     }
 
+    /* add the following styles to ensure proper modal visibility */
+    body.modal-manager-open {
+      overflow: hidden;
+    }
+    modal-manager {
+      display: none;
+    }
+    modal-manager[mode='open'] {
+      display: block;
+    }
+    modal-manager.more-search-facets {
+      --modalWidth: 85rem;
+      --modalBorder: 2px solid #194880;
```

Comment:
QUESTION: Why are we hard-coding colors? Shouldn't we be using using sitewide color conventions?

Reply:
Changed with variables